src/exchange/tradingPair/TradingPairView.scss

```diff
@@ -104,37 +109,70 @@
 
 
 
-// ------------ Pair view -----------------
+// ------------ Active pair view -----------------
 
-.pairView {
+.activePairView {
     display: flex;
     justify-content: space-between;
     align-items: center;
     padding: 12px 26px 12px 12px;
     width: 100%;
 }
 
-.pairViewCurrency {
+.activePairViewIcon {
+    height: 26px;
+    width: 26px;
+    margin-right: 20px;
+    display: flex;
+    align-items: center;
+}
+
+.activePairViewCurrency {
     flex-basis: 50px;
     flex-grow: 1;
 }
 
-.pairViewIcon {
-  height: 26px;
-  width: 26px;
-  margin-right: 20px;
-  display: flex;
-  align-items: center;
+// ------------ Pair view -----------------
+
+.pairView {
+    display: table-row;
+    background-color: #35353E;
+    &:hover {
+        background-color: #393942;
+    }
+    font-size: 14px;
+    & > * {
+      display: table-cell;
+      vertical-align: middle;
+    }
 }
 
-// ---------------------
+.pairViewIcon, .pairViewIconQuote {
+    padding: 5px 0 5px 5px;
+}
+
+.pairViewCurrency {
+    padding: 8px;
+    & + .pairViewCurrency {
```

Comment:
Those kind of selectors are some ninja ones  :) . What do you think about having  
```css
  .pairView {
    .baseIcon {}
    .baseToken {}
    .quoteToken {
       &.dai {
          color: #FFAC13;
       }

       &.weth {
         color: #B15DFF;
       }
    }
    .quoteIcon { }
    .balance {}
  }
}
```